cdk/lib/frontend.ts

```diff
@@ -18,5 +34,384 @@ export class Frontend extends GuStack {
         Stage: GuStageParameter.getInstance(this),
       },
     });
+
+    const app = "frontend";
+
+    const userData = `#!/bin/bash -ev
+    mkdir /etc/gu
+    aws --region ${this.region} s3 cp s3://membership-dist/${this.stack}/${this.stage}/${app}/support-frontend_1.0-SNAPSHOT_all.deb /tmp
+    dpkg -i /tmp/support-frontend_1.0-SNAPSHOT_all.deb
+    /opt/cloudwatch-logs/configure-logs application ${this.stack} ${this.stage} ${app} /var/log/support-frontend/application.log '%Y-%m-%dT%H:%M:%S,%f%z'`;
+
+    const membershipSubPromotionsTable = this.withStageDependentValue({
+      app,
+      variableName: "MembershipSubPromotions",
+      stageValues: {
+        // TODO: Should these dynamo tables be region specific?
+        CODE: "arn:aws:dynamodb:*:*:table/MembershipSub-Promotions-DEV",
+        PROD: "arn:aws:dynamodb:*:*:table/MembershipSub-Promotions-PROD",
+      },
+    });
+
+    const redemptionCodesTable = this.withStageDependentValue({
+      app,
+      variableName: "RedemptionCodes",
+      stageValues: {
+        // TODO: Should these dynamo tables be region specific?
+        CODE: "arn:aws:dynamodb:*:*:table/redemption-codes-DEV",
+        PROD: "arn:aws:dynamodb:*:*:table/redemption-codes-PROD",
+      },
+    });
+
+    const policies = [
+      // TODO: can we 'standardise' the way we load config to use the default permissons from GuEc2App?
+      new GuAllowPolicy(this, "SSMGet", {
+        actions: ["ssm:GetParametersByPath"],
+        resources: [
+          `arn:aws:ssm:${this.region}:${this.account}:parameter/${this.stack}/${app}/${this.stage}`,
+        ],
+      }),
```

Comment:
The `GuEc2App` construct will automatically give us permissions to `CODE/support/frontend` but the library that we use for config stores config in `support/frontend/CODE`

Reply:
Out of interest, what library are you using?

Reply:
I think that [this code](https://github.com/guardian/support-frontend/blob/main/support-frontend/app/wiring/AppLoader.scala#L33) is [overriding](https://github.com/guardian/simple-configuration#custom-ssm-location) the library's [default path](https://github.com/guardian/simple-configuration#configurationloaderload).

Reply:
Ahh so it looks like the default path from `simple-configuration` does align with what's generated for us

Reply:
I'll create a health card to investigate this and see if we can clean it up (I'm guessing we can!)